docs/tools/terraform/reference/cookbook/kafka-debezium-postgres-source.rst

```diff
@@ -0,0 +1,192 @@
+        "connector.class" = "io.debezium.connector.postgresql.PostgresConnector",
+        "snapshot.mode"   = "initial"
+        "database.hostname" : aiven_pg.demo.service_host
+        "database.port" : aiven_pg.demo.service_port
+        "database.password" : aiven_pg.demo.service_password
+        "database.user" : aiven_pg.demo.service_username
+        "database.dbname"           = "defaultdb"
+        "database.server.name"      = "replicator"
+        "database.ssl.mode"         = "require"
+        "include.schema.changes"    = true
+        "include.query"             = true
+        "table.include.list"        = "public.tab1"
+        "plugin.name"               = "wal2json"
+        "decimal.handling.mode"     = "double"
+        "_aiven.restart.on.failure" = "true"
+        "heartbeat.interval.ms"     = 30000
+        "heartbeat.action.query"    = "INSERT INTO heartbeat (status) VALUES (1)"
+      }
+      depends_on = [aiven_service_integration.i1,aiven_kafka_connect.kc,aiven_pg.demo]
+    }
+
+Let's see the different resources we are going to create:
+
+- Version 3.2.1 of the Aiven Terraform provider will be used
+- The PostgreSQL database will be created in "google-europe-north1" cloud provider with a business-4 plan
+- The Aiven Apache Kafka service will be created in "azure-norway-west" cloud and will be configured with a number of properties:
+  
+  - The ``auto_create_topics_enable = true`` property is crucial as it allows the Debezium connector to create the Kafka topics directly.
+  - The ``kafka_connect = false`` property is used because we want to create a separate Aiven for Apache Kafka Connect service.
+
+
+- The Aiven for Apache Kafka Connect service is configured with public access to allow the service to be accessed through a VPC since we are setting up services in different clouds
+- The resource "aiven_service_integration.i1" configures the integration between the AIven for Apache Kafka service and the Aiven for Apache Kafka Connect service. This integration uses 2 internal topics for storing status and offset.
+- The last Aiven service that will be provisioned is the Debezium source connector for PostgreSQL, which is specified by the "connector.class" and is configured with the connection strings to access the PostgreSQL database and listen for all data changes on one or more tables. In our case, the table that is monitored for any new data is "tab1" in "defaultdb", in "public" schema. The plugin used is "wal2json" that converts WAL events (WAL stands for Write Ahead Logging) into JSON payload that is sent to the Kafka topic. The Kafka topic that the Debezium connector creates has the name "replicator.public.tab1", where "replicator" is the logical database used by Debezium connector to monitor for data changes and "public" and "tab1" are the name of the PostgreSQL schema and table name respectively. 
+- The "depends_on" property establishes a dependency between the services creation in order to avoid failures.
```

Comment:
Depends on should not be used most of the time, its better to let TF do its thing, the only place where it is kinda needed ( and thats more a problem with our provider ) is in the connector to tell it to wait for the integration ( its an implicit dependency since creating the integration starts kafka connect up and we have to wait for it)